portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
+function addRandomLanguage() {
+  const languages =
+      ['Español', 'English', 'Deutsch'];
+ 
```

Comment:
It looks like an extra space was added here - can you please remove that? (You can keep the newline here though)

Reply:
I corrected it by deleting the spaces between elements inside the array (I believe that is what you meant, if not can you please give me further explanation?).